src/Items.js

```diff
@@ -39,6 +45,11 @@ class Items extends React.Component {
     super(props);
     this.editItemModeThisLayer = false;
     this.calloutRef = createRef();
+    this.onHeaderClick = this.onHeaderClick.bind(this);
+    this.state = {
+      sortedColumn: 'barcode',
+      sortOrder: 'desc'
```

Comment:
```suggestion
      sortOrder: 'desc',
```
Please, use a comma after each value in the object, if there are more than one